documentation/code-tricks.md

```diff
@@ -9,70 +9,6 @@ Solution: Use a "builder" to construct it for you.
 * [M365_BUILDER](language-reference/domain-modifiers/M365_BUILDER.md)
 * [SPF_BUILDER](language-reference/domain-modifiers/SPF_BUILDER.md)
 
-# Trailing commas
```

Comment:
Problem: Someone sees an old dnsconfig.js with `END` and can't find it in the docs.

Suggestion:

# END

You may see `D()` statements written with `END` at the end.  For example:

{% code title="dnsconfig.js" %}
```javascript
D("example.com", REG_MY_PROVIDER, DnsProvider(DSP_MY_PROVIDER),
  A("foo", "1.2.3.4"),
END);
```
{% endcode %}

As of DNSControl v4.15.0 these `END` statements can be removed.  `END` is a no-op which was added for reasons that no longer matter.  They can be removed.

Reply:
Thank you, Tom, for your feedback! I’ve incorporated it into [this commit](https://github.com/StackExchange/dnscontrol/pull/3248/commits/14d7bd1f07833a2bf14ea866bf5745523bcc3c13). You can preview the updated documentation [here](https://docs.dnscontrol.org/~/revisions/EYzKhdIcHuRVpPYIhxuG/advanced-features/code-tricks#trailing-commas).